EpicLoot/EpicLoot.cs

```diff
@@ -400,6 +403,50 @@ public static void InitializeConfig()
             LoadJsonFile<AdventureDataConfig>("adventuredata.json", AdventureDataManager.Initialize);
             LoadJsonFile<LegendaryItemConfig>("legendaries.json", UniqueLegendaryHelper.Initialize);
             LoadJsonFile<AbilityConfig>("abilities.json", AbilityDefinitions.Initialize);
+            WatchNewPatchConfig();
+        }
+
+        public static void WatchNewPatchConfig()
+        {
+
+            Debug.Log($"Watching For Files");
```

Comment:
Use EpicLoot.Log to avoid log spam

Reply:
Yeah, missed one.  Thought I had caught them all. I'll fix up.